client/src/App.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+import { Route } from 'react-router-dom';
+
+import Modal from './components/UI/Modal/Modal';
+import './App.css';
+import Layout from './components/hoc/Layout/Layout';
+import Home from './containers/Home';
+
+const App = () => {
+  const [showAuthModal, setShowAuthModal] = useState(true);
+
+  return (
+    <>
+      <Layout>
+        <Route
```

Comment:
Aren't we using Browserrouter ? or Any form of Router?

Reply:
@piyushagwl007 yes BrowserRouter check out index.js!
I am just using route to define routes.